CRM/Contribute/BAO/Contribution.php

```diff
@@ -4177,8 +4177,14 @@ private static function getOriginalContribution($contributionID) {
    *
    * @throws \CRM_Core_Exception
    */
-  public static function updateMembershipBasedOnCompletionOfContribution($contributionID, $changeDate) {
+  public static function updateMembershipBasedOnCompletionOfContribution(int $contributionID, string $changeDate) {
```

Comment:
We should also add a deprecated annotation

Reply:
Done